yamllint/cli.py

```diff
@@ -141,6 +141,19 @@ def show_problems(problems, file, args_format, no_warn):
     return max_level
 
 
+def find_project_config_filename(path='.'):
```

Comment:
I see you rename `find_project_config()` to `find_project_config_filename()`, but since it returns a path I would prefer `find_project_config_path()`, what do you think?

Reply:
Valid point. I agree. Corrected, squashed, and pushed.